css/properties/scrollbar-gutter.json

```
@@ -22,8 +22,14 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
-              "impl_url": "https://webkit.org/b/167335"
+              "version_added": "17",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "CSS scrollerbar-gutter property",
+                  "value-to-set": "true"
```

One last little detail, looks like!

```suggestion
                  "value_to_set": "true"
```

---

Soga!